packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Basic.Example.tsx

```diff
@@ -1,79 +1,103 @@
+  },
+  {
+    key: 'linkWithOnClick',
+    name: 'Link click',
+    href: 'http://bing.com',
+    onClick: (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => {
+      alert('Link clicked');
+      ev.preventDefault();
+    },
+    target: '_blank'
+  },
+  {
+    key: 'disabled',
+    text: 'Disabled item',
+    disabled: true,
+    onClick: () => console.error('Disabled item should not be clickable.')
+  }
+];
+
+export class ContextualMenuBasicExample extends React.Component<{}, IContextualMenuBasicExampleState> {
+  private contextualMenuTarget = React.createRef<HTMLAnchorElement>();
 
-export class ContextualMenuBasicExample extends React.Component {
   constructor(props: {}) {
     super(props);
     this.state = {
-      showCallout: false
+      showContextualMenu: false
     };
   }
 
   public render(): JSX.Element {
+    const { showContextualMenu } = this.state;
+
     return (
       <div>
-        <DefaultButton
-          text="Click for ContextualMenu"
-          menuProps={{
-            shouldFocusOnMount: true,
-            items: [
-              {
-                key: 'newItem',
-                text: 'New',
-                onClick: () => console.log('New clicked')
-              },
-              {
-                key: 'divider_1',
-                itemType: ContextualMenuItemType.Divider
-              },
-              {
-                key: 'rename',
-                text: 'Rename',
-                onClick: () => console.log('Rename clicked')
-              },
-              {
-                key: 'edit',
-                text: 'Edit',
-                onClick: () => console.log('Edit clicked')
-              },
-              {
-                key: 'properties',
-                text: 'Properties',
-                onClick: () => console.log('Properties clicked')
-              },
-              {
-                key: 'linkNoTarget',
-                text: 'Link same window',
-                href: 'http://bing.com'
-              },
-              {
-                key: 'linkWithTarget',
-                text: 'Link new window',
-                href: 'http://bing.com',
-                target: '_blank'
-              },
-              {
-                key: 'linkWithOnClick',
-                name: 'Link click',
-                href: 'http://bing.com',
-                onClick: (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => {
-                  alert('Link clicked');
-                  ev.preventDefault();
-                },
-                target: '_blank'
-              },
-              {
-                key: 'disabled',
-                text: 'Disabled item',
-                disabled: true,
-                onClick: () => console.error('Disabled item should not be clickable.')
-              }
-            ]
-          }}
+        This example directly uses ContextualMenu to show how it can be attached to arbitrary elements. The remaining examples use
```

Comment:
Just thinking out loud but should this explanation also be in overview? Maybe in something like a "Notes" section?

Reply:
Hmm, I think it's better with the example, in case someone wants to change the example, or wonders why it's there. I think having this specific example NOT be a button helps call it out, too.

Reply:
Probably not. The overview is really just meant for a high-level explanation (the components using it as a dumping ground for extra info ideally shouldn't be doing that). Right now the best place for content which is this developer-specific is next to the relevant example.

Reply:
Sounds good, thanks everybody!